docs/components/api-item.js

```diff
@@ -0,0 +1,178 @@
+        return `#${slugger.slug(namespace)}`;
+    });
+
+const formatters = createFormatters(linkerStack.link);
+
+class ApiItem extends React.Component {
+
+    md = (ast, inline) => {
+        if (inline && ast && ast.children.length && ast.children[0].type === 'paragraph') {
+            ast = {
+                type: 'root',
+                children: ast.children[0].children.concat(ast.children.slice(1))
+            };
+        }
+        return <span dangerouslySetInnerHTML={{__html: formatters.markdown(ast)}}/>;
+    }
+    formatType = type => {
+        return <span dangerouslySetInnerHTML={{__html: formatters.type(type)}}/>;
+    }
+
+    render() {
+        const section = this.props;
+
+        const empty = (members) => !members || members.length === 0;
+
+        const membersList = (members, title) => {
+            return !empty(members) &&
+                <div>
+                    <div className='py6 color-gray mt12 txt-l'>{title}</div>
+                    <div className='clearfix'>
+                        {members.map((member, i) => <ApiItemMember key={i} {...member}/>)}
+                    </div>
+                </div>;
+        };
+
+        return (
+            <section className='px12 py12 clearfix prose mb24 bg-white contain'>
+                {!this.props.nested &&
+                    <div className='mb24 clearfix'>
+                        <h2 className='mb12' id={section.namespace.toLowerCase()}><a className="unprose color-blue-on-hover" href={`#${section.namespace.toLowerCase()}`}>{section.name}</a></h2>
+                        {section.context && section.context.github &&
+                            <a className='pt6 color-gray color-gray-dark-on-hover txt-s txt-bold unprose' href={section.context.github.url}><span className="inline-block mb-neg6 mr6"><Icon name="github" /></span><span>{section.context.github.path}</span></a>}
+                    </div>}
+
+                {this.md(section.description)}
+
+                {!empty(section.augments) &&
+                    <p>Extends {section.augments.map((tag, i) =>
+                        <span key={i} dangerouslySetInnerHTML={{__html: formatters.autolink(tag.name)}}/>)}.</p>}
+
+                {section.kind === 'class' &&
+                    !section.interface &&
+                    (!section.constructorComment || section.constructorComment.access !== 'private') &&
+                    <div className='txt-code px6 py6 txt-s round bg-gray-faint'
+                        dangerouslySetInnerHTML={{__html: `new ${section.name}${formatters.parameters(section)}`}}/>}
+
+                {section.version && <div>Version: {section.version}</div>}
+                {section.license && <div>License: {section.license}</div>}
+                {section.author && <div>Author: {section.author}</div>}
+                {section.copyright && <div>Copyright: {section.copyright}</div>}
+                {section.since && <div>Since: {section.since}</div>}
```

Comment:
what are these parts for? I'm not seeing them when I run the page

Reply:
Hmmm I'm not sure either actually. This is currently included here https://github.com/mapbox/mapbox-gl-js/blob/mb-pages/docs/pages/api.js#L197-L202.